pkgs/tools/security/hash-identifier/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitLab, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "hash-identifier";
+  version = "1.2";
+
+  src = fetchFromGitLab {
+    owner = "kalilinux";
+    repo = "packages/hash-identifier";
+    rev = "20ebdd2f33ffc1bdcc010ff582be203696b0bb2e";
```

Comment:
```suggestion
    rev = "kali/1.2+git20180314-0kali1";
```

Reply:
Oops, you're correct, I could've swarn I did it that way to begin with though..